examples/nifgen_standard_function/_nifgen_helpers.py

```diff
@@ -0,0 +1,40 @@
+"""nifgen Helper classes and functions for MeasurementLink examples."""
+
+from typing import Any, Dict, Optional
+
+import grpc
+import nifgen
+
+import ni_measurementlink_service as nims
+
+# To use a physical NI FGen instrument, set this to False or specify
```

Comment:
The correct driver name is "NI-FGEN" with a dash and in all uppercase.

Reply:
Updated